signer/src/stacks/contracts.rs

```diff
@@ -1029,6 +1030,10 @@ pub enum WithdrawalRejectErrorMsg {
     /// been either accepted or rejected already.
     #[error("the smart contract has been updated to indicate that the request has been completed")]
     RequestCompleted,
+    /// A withdrawal request is still active if it's reasonably possible
```

Comment:
Suggestion: maybe tweak this description or the `RequestBeingFulfilled` one to just make it more clear the differences; "active" and "being fulfilled" felt a bit like the same thing to me when I first glanced this over.

Reply:
Yeah, I thought the same thing and I couldn't think of a good name. Hopefully we can kill this soon by updating the smart contract.